ext/fluxnet_simulations/data_processing.jl

```diff
@@ -143,6 +157,32 @@ function get_data_at_start_date(
     return preprocess_func(v[idx_start])
 end
 
+"""
+    read_fluxnet_data(site_ID; hour_offset_from_UTC = nothing)
+
+Reads Fluxnet CSV data and processes timestamps, converting local time to UTC.
+Returns (data, columns, local_datetime, UTC_datetime).
+
+If `hour_offset_from_UTC` is nothing, the UTC datetime is not returned.
+"""
+function read_fluxnet_data(site_ID; hour_offset_from_UTC = nothing)
+    fluxnet_csv_path = ClimaLand.Artifacts.experiment_fluxnet_data_path(site_ID)
+    (data, columns) = readdlm(fluxnet_csv_path, ','; header = true)
+
+    # Convert timestamps
+    local_datetime = DateTime.(string.(Int.(data[:, 1])), "yyyymmddHHMM")
```

Comment:
it might be safer to extract the column with the label TIMESTAMP instead of indexing with 1

we should also note that this is what the function does (since we sometimes want to return a different timestamp)